tests/lib/rules/no-private-routing-service.js

```diff
@@ -38,6 +38,9 @@ ruleTester.run('no-private-routing-service', rule, {
       }
     }
     `,
+    'export default Component.extend({ someService: service() });',
+    'export default Component.extend({ notAService: "a value" });',
+    'export default Component.extend({ anInt: 25 });',
```

Comment:
Can you group all the native class examples together and all the classic class examples together?